### ISSUE TITLE
nxinspect crashes when external link file is missing 

### ISSUE THREAD

**Issue by jkotan** [nexpy/nxvalidate#12]
Currently, if an external link file is missing  the nxinspect crashes.
```
  File "/usr/lib/python3/dist-packages/nexusformat/nexus/tree.py", line 440, in __init__
    raise NeXusError(f"'{self._filedir}/' is not accessible")
nexusformat.nexus.tree.NeXusError: '/home/jkotan/gpfs/current/raw/eigesv1_01309/eiger9m/' is not accessible
```
Instead It would be good to print the error and validate other parts of the file.

**Comment by rayosborn** [comment]
I agree that this needs fixing.

**Comment by rayosborn** [comment]
This should be fixed by a47442469ee1c2a5fd0cb3a4b4966326b6a5345e

**Comment by jkotan** [comment]
Thanks.  The nxinspect  does not crash anymore.
